spec/factories/course_excluded_exercises.rb

```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :course_excluded_exercise do
+    sequence_number { Kernel::rand(10) }
+    excluded_uuid   { SecureRandom.uuid.to_s }
+    course_uuid     { SecureRandom.uuid.to_s }
```

Comment:
Totally cosmetic: I'd move this up to the top so the sequence number scope is more obvious.